workspace/spec/lita/handlers/teamcity_spec_anki.rb

```diff
@@ -471,4 +471,41 @@
 
   end
 
+  describe '#list_cps_to_commit' do
+    repo_name = "build-tools"
+    repos_dir = File.join(Dir.home, 'repos')
+    repo_uri  = "git@github.com:anki/#{repo_name}.git"
```

Comment:
Replace company-specific-info

Reply:
I will update all company-specific-info after [PR #BI-1413](https://ankiinc.atlassian.net/browse/BI-1413) is approved and merge